src/client/components/DetailsModal/Details/DanishHouseDetails.tsx

```diff
@@ -0,0 +1,126 @@
+      )
+      .required(textKeys.GENERIC_ERROR_INPUT_REQUIRED()),
+    data: Yup.object({
+      street: Yup.string().required(textKeys.GENERIC_ERROR_INPUT_REQUIRED()),
+      zipCode: Yup.string().matches(
+        /^[0-9]{4}$/,
+        textKeys.GENERIC_ERROR_INPUT_FORMAT(),
+      ),
+      livingSpace: Yup.number()
+        .typeError(textKeys.GENERIC_ERROR_INPUT_REQUIRED())
+        .min(1, textKeys.GENERIC_ERROR_INPUT_FORMAT())
+        .required(textKeys.GENERIC_ERROR_INPUT_REQUIRED()),
+      householdSize: Yup.number()
+        .min(1, textKeys.GENERIC_ERROR_INPUT_FORMAT())
+        .required(textKeys.GENERIC_ERROR_INPUT_REQUIRED()),
+      isYouth: Yup.boolean(),
+      extraBuildings: Yup.array().of(
+        Yup.object().shape({
+          type: Yup.string().required(),
+          area: Yup.number()
+            .min(1, textKeys.GENERIC_ERROR_INPUT_FORMAT())
+            .required(textKeys.GENERIC_ERROR_INPUT_REQUIRED()),
+          hasWaterConnected: Yup.boolean().required(),
+        }),
+      ),
+    }).required(textKeys.GENERIC_ERROR_INPUT_REQUIRED()),
+  })
+}
+
+export type DanishHouseDetailsProps = {
+  formikProps: FormikProps<QuoteInput>
+}
+
+export const DanishHouseDetails = ({
+  formikProps,
+}: DanishHouseDetailsProps) => {
+  const { birthDate: birthDateFormat } = useCurrentLocale()
+
+  return (
+    <Content>
+      <ContentRow>
+        <ExtraBuildingsInput market="DK" formikProps={formikProps} />
+      </ContentRow>
+      <ContentColumn>
+        <InputGroup>
+          <TextInput
+            name="firstName"
+            label="DETAILS_MODULE_TABLE_FIRSTNAME_CELL_LABEL"
+            formikProps={formikProps}
+          />
+          <TextInput
+            name="lastName"
+            label="DETAILS_MODULE_TABLE_LASTNAME_CELL_LABEL"
+            formikProps={formikProps}
+          />
+          <BirthDateInput
+            name="birthDate"
+            placeholder={birthDateFormat.backendFormatExample}
+            formikProps={formikProps}
+          />
+        </InputGroup>
+      </ContentColumn>
+      <ContentColumn>
+        <InputGroup>
+          <TextInput
+            name="data.street"
+            label="DETAILS_MODULE_TABLE_ADDRESS_CELL_LABEL"
+            formikProps={formikProps}
+          />
+          <ZipcodeInput
+            name="data.zipCode"
+            market="NO"
+            formikProps={formikProps}
+          />
+          <AreaInput
+            name="data.livingSpace"
+            label="DETAILS_MODULE_TABLE_SIZE_CELL_LABEL_APARTMENT"
+            formikProps={formikProps}
+          />
+          <DetailInput
+            name="data.householdSize"
+            field={{
+              label: 'DETAILS_MODULE_TABLE_HOUSEHOLD_SIZE_CELL_LABEL',
+              placeholder: '',
+              type: inputTypes.number,
+            }}
+            formikProps={formikProps}
+          />
+          <BooleanInput
+            name="data.isStudent"
```

Comment:
This one isn't in the yup object above?

Reply:
No. This comes from formik. We'r initialising it [here](https://github.com/HedvigInsurance/web-onboarding/blob/master/src/client/components/DetailsModal/index.tsx#L220)